[VPD-485]: Add Yield+ (RelativePositionManager) documentation (#343)
VenusProtocol/venus-protocol-documentation: whats-new/yield-plus.md

Capital utilization tells you how much of your deposited collateral is currently locked by your open position. The remaining portion — **Available Capital** — can be used to open additional positions or withdrawn.

## How It Works

### Step 1 — Activate and Open

Deposit your DSA collateral, select a trading pair, set your leverage, and submit in one transaction. Behind the scenes, the system:

1. Deploys a dedicated PositionAccount for this pair (first time only)
2. Enters the DSA market and supplies your collateral
3. Flash loans the short asset via LeverageStrategiesManager
4. Swaps the short asset to the long asset
5. Supplies the long asset to Venus, establishing the leveraged position

### Step 2 — Monitor and Manage

Once open, your position generates yield automatically:

- **Supply APY** on the long asset
- **DSA APY** on your collateral
- minus **Borrow APY** on the short asset

You can monitor Health Factor, PnL, entry price, and liquidation price at any time. Available management actions include increasing the position, supplying additional collateral, proportionally reducing the position, withdrawing unused collateral, or fully closing and deactivating.

> ### Step 3 — Close

Reviewer: ```suggestion
### Step 3 — Reduce and Exit
```

> Closing is proportional — you specify what fraction of the position to close (1% to 100%) in a single transaction. Profits are automatically converted into DSA collateral. Losses are covered by your existing DSA collateral.

Reviewer: ```suggestion
Reducing is proportional — you specify what fraction of the position to close (1% to 100%) in a single transaction. Profits are automatically converted into DSA collateral. Losses are covered by your existing DSA collateral.
```

> After a full close, the position account remains active for re-entry. Use **Deactivate** (Exit Market) to fully withdraw and shut down the position account.

Reviewer: ```suggestion
After a full reduce(100% reduce), the position account remains active for re-entry. Use **Deactivate** (Exit Market) to fully withdraw and shut down the position account.
```

> 

Reviewer: ```suggestion

We also provide a one-click Exit function that completes both "reduce 100%" and "Exit Market" in a single transaction.

```
